### ISSUE TITLE
[Masthead v2.0.0]: bugs identified in 2.0.0 (Phase 1)

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#9310]
**Engineering info:**  
- Innovation Team engineer: @jkaeser 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/ADCMS-2570

-----

### Description

List of bugs identified by design QA

### Component(s) impacted

Masthead v2

### Browser

Chrome

### Carbon for IBM.com version

v2 feature branch

### Application/website

Carbon for AEM

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/masthead-v2/web-components/?path=/story/components-masthead--default

### Steps to reproduce the issue (if applicable)

reduce browser width to anything between 800–959px wide

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by proeung** [comment]
@oliviaflory Thanks for opening this issue. Is this the only feedback/bug that you've noticed as part of this design QA ticket (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/9307) or are there more items that you've spotted? If it's the latter, can we capture everything into a single issue/ticket instead of individual ones?

**Comment by proeung** [comment]
@oliviaflory Got it and no worries!

Let's use this issue to capture all of the styling bugs that you and @JaniceChristineM discovered and leave this one (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/9307) for the design QA task, which is assigned to you and Janice for this sprint. Sounds good?

**Comment by oliviaflory** [comment]
## Style bugs L0

Menu item and caret are two different colors – they should be the same. 
See tokens listed in [Spec sheet](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=1970%3A110816)
- [ ] enabled fixed
- [ ] hover fixed
<img width="894" alt="Screen Shot 2022-08-29 at 6 02 07 PM" src="https://user-images.githubusercontent.com/45692622/187463668-8e1a2c3a-8a09-4507-8211-bac94acb908f.png">

---

Chat and Login icons are a different color than the Search icon
See tokens listed in [Spec sheet](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=1970%3A110816)
- [ ] enabled fixed
- [ ] hover fixed
<img width="539" alt="Screen Shot 2022-08-29 at 6 02 32 PM" src="https://user-images.githubusercontent.com/45692622/187479044-73261035-e07a-4200-bce7-7faf8da1347c.png">

--- 

Focus state for all menu items and menu icons should not have any background color
See states and tokens listed in [Spec sheet](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=1970%3A110816)
- [ ] menu items fixed
- [ ] menu icons fixed
- [ ] IBM logo fixed

--- 

I don't see an option or knob for the Optional Cluster name
Apologies I cannot remember if this was removed from the scope of the September launch
[See specs](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=2177%3A120058)

---
## Style bugs mega menu

H1 hover and active state text colors are incorrect (hover was mislabeled in spec sheet, but is now updated)
See [Spec sheet](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=2199%3A122077)
- [ ] hover fixed
- [ ] active state fixed

H3 hover and active state text colors are incorrect
See [Spec sheet](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=2199%3A122077)
- [ ] hover fixed
- [ ] active state fixed

**Comment by oliviaflory** [comment]
## Style bugs mobile

_TBD_

**Comment by oliviaflory** [comment]
## Mega menu

Tiered menu tab order is horizontal, should be top to bottom 
[See functional specs](https://www.figma.com/file/35JfN0Sg4ltIIjM7jBDKMk/Navigation?node-id=2325%3A136628)

![tab order Aug-30-2022 12-32-03](https://user-images.githubusercontent.com/45692622/187491269-ce022185-073f-4697-a3ee-c507ce9dec3a.gif)
<img width="1024" alt="Screen Shot 2022-08-30 at 12 34 07 PM" src="https://user-images.githubusercontent.com/45692622/187491529-71d35e02-d7a3-45bd-a4d8-cb93bce674d6.png">

**Comment by jkaeser** [comment]
Hi, @oliviaflory, and thank you for the detailed feedback! I've addressed most of it over in #9324, but I'd argue that some of the changes are out of scope for this round of work.

1. No changes to the mobile navigation are in scope, though I did remove the extra dark border since it was a quick fix. I looked into updating the color tokens, but that part of the code needs a refactor to pay back some tech debt, which is enough work for a separate ticket.
2. There are some ramifications I'll need to bring up with the team before we can changing the tabbing order for the tabbed megamenu layout, specifically dealing with the source order of the nav items over in the translations repository.

I'm happy to discuss further if you'd like. Otherwise, let's capture these two pieces of feedback in separate tickets so we can address them in one of the next phases.
